corehq/messaging/scheduling/models/content.py

```diff
@@ -139,6 +141,18 @@ def send(self, recipient, logged_event, phone_entry=None):
             return
 
         send_mail_async.delay(subject, message, settings.DEFAULT_FROM_EMAIL, [email_address])
+
+        email = Email(
+            domain=logged_event.domain,
+            date=datetime.utcnow(),
+            couch_recipient_doc_type=logged_event.recipient_type,
+            couch_recipient=logged_event.recipient_id,
+            messaging_subevent_id=logged_event.pk,
```

Comment:
I think you need to use `logged_subevent` here instead?

Reply:
Good catch, the pks just happened to be the same thing on mine since I was only sending one email each time.